v2/teams.ts

```diff
@@ -0,0 +1,157 @@
+import type {
+  TeamsResultPage, TeamFields, TeamsIncluded, Team,
+  TeamMembership, TeamLink, TeamPermissionSetting
+} from "./lib/identity.ts";
+
+type TODO = unknown;
+
+// Common API client contract
+interface ApiClient {
+  fetchJson(opts: {
+    method: "GET" | "POST" | "DELETE";
```

Comment:
It looks like this is the first DELETE call in this project, and so the new method needs to be also added to `client.ts`:
https://github.com/cloudydeno/deno-datadog_api/blob/657811ec549cd4c16a5cdb096c07118aff45244e/client.ts#L40